packages/edit-post/src/components/manage-blocks-modal/category.js

```diff
@@ -23,18 +25,32 @@ function BlockManagerCategory( {
 	toggleVisible,
 	toggleAllVisible,
 } ) {
-	if ( ! blockTypes.length ) {
+	const settings = useContext( EditPostSettings );
+	const { allowedBlockTypes } = settings;
```

Comment:
These are all available in the editor store why not just use that store instead of adding a new context provider?

Reply:
Hi @youknowriad, when a user disables a block in the block manager the block is removed from the editor store allowed blocks. Using just the editor store, it is not possible to differentiate between a block disabled by the user and a block restricted on the server using a plugin. In this case, we need to distinguish both cases.

Reply:
It's minor but It feels like something that can be solved in the store somehow instead of adding a new context for sharing data while we already have a context for that (the store).